test/services/lessons/addMaterialUnit.test.js

```diff
@@ -0,0 +1,38 @@
+const assert = require('assert');
```

Comment:
Codacy found an issue: ['assert' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4977295)